L/LLVM/common.jl

```diff
@@ -262,19 +312,60 @@ function configure(ARGS, version)
         push!(products, ExecutableProduct("llvm-mca", :llvm_mca, "tools"))
     end
 
-    name = "LLVM"
+    name = "LLVM_full"
     config = "LLVM_MAJ_VER=$(version.major)\n"
     if assert
         config *= "ASSERTS=1\n"
         name = "$(name)_assert"
     end
-    return name, sources, config * buildscript, products
+    # Dependencies that must be installed before this package can be built
+    # TODO: Zlib, LibXML2
+    dependencies = []
```

Comment:
```suggestion
    dependencies = Dependency[]
```
to use the new type for the dependencies